packages/aws-cdk-lib/aws-cloudwatch/lib/metric.ts

```diff
@@ -259,6 +259,33 @@ export class Metric implements IMetric {
     });
   }
 
+  /**
+ * Make a new Alarm for this metric using anomaly detection
+ *
+ * @param scope The scope in which to create the alarm
+ * @param id The ID of the alarm
+ * @param props Alarm creation properties for anomaly detection
+ * @param metric The metric to create the alarm for
+ * @returns The newly created Alarm
+ */
+  public static createAnomalyDetectionAlarmFromMetric(scope: Construct, id: string, props: CreateAnomalyDetectionAlarmProps, metric: IMetric): Alarm {
+    if (!Alarm.isAnomalyDetectionOperator(props.comparisonOperator)) {
+      throw new Error(`Invalid comparison operator for anomaly detection alarm: ${props.comparisonOperator}`);
+    }
+    const anomalyDetectionExpression = new MathExpression({
+      expression: `ANOMALY_DETECTION_BAND(m0, ${props.bounds ?? 1})`,
```

Comment:
since `bounds` is a required property, do we need null check here?

Reply:
Right, this is useless now that the property is required.

Reply:
Or should we have a default value for it?